docs/QUICK_START.md

```diff
@@ -0,0 +1,201 @@
+# mLRS Documentation: Quick Start Guide #
+
+([back to main page](../README.md))
+
+This page provides hints, suggestions and links to help get new users started with mLRS quickly.
+
+This guide covers the use case where mLRS is used to provide a single radio link to carry both RC control and full bi-directional MAVLink Telemetry.  When we discuss configuration of your flight controller, we assume you are running ArduPilot.  We also assume you use EdgeTX or OpenTX and that you will connect a ground station and/or use the [Yaapu Telemetry App](CRSF.md).
+
+We expect that you are starting fresh and that all ArduPilot and mLRS parameters not covered here still have their default settings.
+
+The steps here should be followed in order and when practical we provide a way to confirm a step was completed successfully.
+
+Some notation and terminology:
+- We use "Tx module" and "receiver" to refer to a mLRS module in the Tx/transmitter role and Rx/receiver role. In the use case assumed here, the Tx module will be installed in a EdgeTX or OpenTX radio handset, and the receiver will be mounted on the vehicle and connected to the flight controller.
+- We use lower case "rx" and "tx" when referring to the receive and transmit pads or pins of a serial port connection.
+
+## 1. Choose and purchase a Tx module and a compatible receiver
+
+There are many supported choices of mLRS hardware, including off-the-shelf options and  many DIY designs.
+We cover here three popular but mutually incompatible commercially available off-the-shelf options.  We summarize some pros and cons below to help you choose. Don't mix Tx module and receiver between these options.
+
+### MatekSys mLRS [mR900-30-TX](https://www.mateksys.com/?portfolio=mr900-30-tx) module kit with their [mR900-30](https://www.mateksys.com/?portfolio=mr900-30) receiver
+- These 900 MHz band modules were designed specifically for mLRS and arrive pre-flashed with mLRS.
+- This option should provide the longest range of the three discussed here.
+- The mR900-30-TX module kit includes an HC04 module for a convenient Bluetooth serial connection to your ground station software running on a computer or Android device.
+- These modules are especially easy to flash via DFU over their USB-C connection.
+- These use the SX126x LoRa chip and support 19 Hz, 31 Hz and FSK(50 Hz) modes.
+- This option does not work with any ELRS receivers.
+- The Tx module kit does not include an antenna or a JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
```

Comment:
some vendors actually sell cases ... just found it for rotorsme, but recall also having once seen for a danish (?) vendor and very few others
not sure, but maybe worth mentioning that some vendors sell them (would be a better option than 3d printer serive probably)